src/wolfscp.c

```diff
@@ -1104,6 +1106,7 @@ int ParseScpCommand(WOLFSSH* ssh)
                             /* skip space */
                             idx += 2;
                             ssh->scpBasePath = cmd + idx;
+                            clean_path((char*)ssh->scpBasePath);
```

Comment:
clean_path will make adjustments to the buffer (always smaller than starting string if changed). Since this is a pointer to cmd + idx it will alter the buffer cmd. I did not see any issue with it but is worth mentioning.